.github/workflows/validate_version.yaml

```diff
@@ -0,0 +1,24 @@
+name: Validate Release
+
+on:
+  deployment:
```

Comment:
why on deployment? who triggers it?
not better using workflow dispatch?

Reply:
@diabawada only `pull_request` required here

Reply:
fixed